src/components/Filters/Filters.tsx

```diff
@@ -171,9 +179,14 @@ class Filters extends React.Component<ComposedProps, State> {
             type="button"
             aria-controls={collapsibleID}
             aria-expanded={filterIsOpen}
+            disabled={disabled}
           >
             <div className={styles.FilterTriggerLabelContainer}>
-              <h2 className={styles.FilterTriggerTitle}>{filter.label}</h2>
+              <h2 className={styles.FilterTriggerTitle}>
+                <TextStyle {...filterTriggerTitleTextStyle}>
```

Comment:
This would be a little less code
`<TextStyle  variation={disabled ? VariationValue.Subdued : undefined}>`